doc/how_to_introduce_breaking_changes.md

```diff
@@ -0,0 +1,198 @@
+# How to introduce Breaking Changes in service command
+
+Azure CLI has bi-annual breaking change releases coinciding with Microsoft **Build** and **Ignite**. Limiting breaking changes to twice a year provides a stable experience for customers while being able to keep up to date with the latest versions of Azure CLI and plan accordingly for announced breaking changes.
+
+## Breaking Changes in Azure CLI
+
+A breaking change refers to a modification that disrupts backward compatibility with previous versions. The breaking changes could cause a customer's script or automation written in a previous version to fail.
+
+The common examples of breaking changes include:
+* Modifying the names of parameters/commands.
+* Modifying the input logic of parameters.
+* Modifying the format or properties of result output.
+* Modifying the current behavior model.
+* Adding additional verification that changes CLI behavior.
+
+To mitigate the impact of breaking changes, Azure CLI delays breaking changes and coordinates half-yearly **Breaking Change Releases** that bundle multiple breaking changes together. This approach helps users plan ahead and adapt to the modifications effectively.
+
+### Breaking Change Window
+
+The breaking change window *allows* for service command breaking changes. When a Pull Request is merged during this sprint, it will be included in the next Breaking Change Release.
+
+The timing of the breaking change window in Azure CLI aligns with [Microsoft Build](https://build.microsoft.com/) and [Microsoft Ignite](https://ignite.microsoft.com/). You could find the next Breaking Change Release plan in our [milestones](https://github.com/Azure/azure-cli/milestones).
+
+> If you would like to release ad-hoc breaking changes, reach out to the CLI team to provide an explanation for the necessity of these changes. The exceptions can be provide in the following cases:
+> * The critical bugs need hotfix
+> * The security patch
+> * If server side has produced a breaking change which is inevitable for users, then CLI side has to adapt it
+> 
+> The above situation makes it necessary to introduce breaking change as soon as possible.
+
+### Pre-announce Breaking Changes
+
+All breaking changes **must** be pre-announced several sprints ahead Release. There are two approaches to inform both interactive users and automatic users about the breaking changes.
+
+1. (**Mandatory**) Breaking Changes must be pre-announced through Warning Log while executing.
+2. (*Automatic*) Breaking Changes would be collected automatically and listed in [Upcoming Breaking Change](https://learn.microsoft.com/en-us/cli/azure/upcoming-breaking-changes).
+
+## Workflow
+
+### Overview
+
+* CLI Owned Module
+  * Service Team should create an Issue that requests CLI Team to create the pre-announcement several sprints ahead Breaking Change Window. The CLI team will look at the issue and evaluate if it will be accepted in the next breaking change release.
```

Comment:
Perhaps we can consider adding the tag of `breaking-change request` to this type of issue, which can make it easier for us to pay attention to these important issues in the vast number of issues.

Reply:
> create an Issue 

Additionally, it is recommended to provide a link of the corresponding feature request template here

Reply:
> Perhaps we can consider asking service team to add the tag of `breaking-change request` to this type of issue

In the current [Implementing Breaking Changes Doc](https://eng.ms/docs/cloud-ai-platform/azure-core/azure-management-and-platforms/control-plane-bburns/azure-cli-tools-azure-cli-powershell-and-terraform/azure-cli-tools/teams_docs/azcli_docs/breaking_changes), the created issue should be attached with `Breaking Change` label. I think we could follow that design.

Reply:
like the tag of breaking-change, it's worth having it and align with services to start using.

Reply:
Add a sentence
> The issue should include the label Breaking Change.